### DISCUSSION TITLE
데이터 추출에 대한 질문드립니다.

### DISCUSSION THREAD

**Discussion by kkgcer** [changdoc/naver-vaccine-macro#215] [Q&A]
graphql result 이 칸에 데이터는 집어넣긴했는데

최대 100개까지만 들어가는건가요?

네이버측에서 100개까지만 검색을 하는진 모르겠는데 스크립트 넣고 병원등록하면 칸에 100개밖에 안떠서
100개만 필터하는지 아니면 지도상에 보이는 전체를 필터하는지 여쭤봅니다.

그리고 혹시, 죄송하지만 혹시 카카오는 pc에서 지원하지 않는건가요? 네이버는 사측에서 그러는건지 30분마다 시간이밀려서 
쿠키를 지워야하는거같은데. 혹시나 카카오도 같이해보려하는데 pc에서 지원하는지 모르겠어서 같이 여쭤봅니다..

**Comment by kkgcer** [comment]
자문자답이지만 100개에 대한 내용은 아래 q&a를 보고 찾았습니다. 카카오가 pc에서 지원하는지만 여쭤봅니다..

**Comment by changdoc** [comment]
pc 지원이 되는걸로는 알고 있는데 제가 파악해서 예약 시도 하는 부분까진 구현 하진 않았고 아직은 구현 계획이 없습니다.
